roles/swift-ring/tasks/main.yml

```diff
@@ -13,3 +13,4 @@
   command: sudo -u swiftops /usr/local/bin/swifttool -i /home/swiftops/.ssh/id_rsa
            bootstrap --config /etc/swift/ring_definition.yml --outdir /etc/swift
   when: not should_boostrap|success
+  notify: restart swift services
```

Comment:
yeah, the action on the ring won't trigger a restart on the other nodes. I still am curious why we aren't running into this in our deploys. But it really does sound like we need to change the site.yml order so that the rings get built before the services are started. Swift is supposed to re-read the ring timestamp from time to time and react to changes without restart, so maybe you could show more of the error you're seeing?